oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/VersionGarbageCollector.java

```diff
@@ -1770,6 +1770,21 @@ public void removeGarbage(final VersionGCStats stats) {
                         // fix for sonar : converted to long before operation
                         detailedGCStats.documentsUpdateSkipped((long)oldDocs.size() - updatedDocs);
                     }
+                } else {
+                    // collect approx stats only in case of dryRun by assuming everything would succeed
+
+                    // for deleted properties, oldRevs & unmergedBC
+                    stats.updatedDetailedGCDocsCount += updateOpList.size();
+                    stats.deletedPropsCount += deletedPropsCountMap.values().stream().reduce(0, Integer::sum);
+                    stats.deletedInternalPropsCount += deletedInternalPropsCountMap.values().stream().reduce(0, Integer::sum);
+                    stats.deletedPropRevsCount += deletedPropRevsCountMap.values().stream().reduce(0, Integer::sum);
+                    stats.deletedInternalPropRevsCount += deletedInternalPropRevsCountMap.values().stream().reduce(0, Integer::sum);
+                    stats.deletedUnmergedBCCount += deletedUnmergedBCSet.size();
+
+                    // for orphan nodes.
+                    stats.updatedDetailedGCDocsCount += orphanOrDeletedRemovalMap.size();
+                    stats.deletedDocGCCount += orphanOrDeletedRemovalMap.size();
+//                    stats.deletedOrphanNodesCount += orphanOrDeletedRemovalMap.size();
```

Comment:
Unless there's a reason to keep this comment, I would probably remove it

Reply:
On local, I do have stats for orphan nodes. I had to comment it out to make it compile.